src/lib.rs

```diff
@@ -152,6 +152,10 @@ macro_rules! __if_chain {
             $($pat)|+ => __if_chain! { @expand $other $($tt)+ }
         }
     };
+    (@expand $other:block let $ident:ident:$ty:ty = $expr:expr; $($tt:tt)+) => {
+        let $ident:$ty = $expr;
```

Comment:
I'd prefer some extra spaces, for clarity:

```rust
    (@expand $other:block let $ident:ident: $ty:ty = $expr:expr; $($tt:tt)+) => {
        let $ident: $ty = $expr;
```

Reply:
Done.